### PR TITLE
Submit for SketchExporter

### PR DESCRIPTION
**Pull request by gliyao**
Hi, this plugin help user export their .sketch assets to Images.xcassets. But it is depend on cli sketchtool.

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -830,6 +830,12 @@
         "name": "PrettyPrintJSON",
         "url": "https://github.com/psobko/PrettyPrintJSON",
         "description": "Pretty print JSON from the console."
+      },
+      {
+        "name": "SketchExporter",
+        "url": "https://github.com/gliyao/SketchExporter",
+        "description": "Export AppIcon and icons to your Images.xassets (require sketchtool - cli tool for Sektch)",
```
Comment (jurre) [inline packages.json]: You misspelled Sketch there at the end I think :)

**Comment by jurre** [comment]
Nice one, does the plugin warn you if you haven't installed the CLI dependency?

File: packages.json (hunk 1 above)
Comment (gliyao) [inline packages.json]: Oh, yes.

**Comment by gliyao** [comment]
I check cli sketchtool at do_sketch.sh first line. 
If user not install sketchtool it will echo "I require sketchtool but it's not installed.  Aborting."

**Pull request merged**